mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg

```diff
@@ -21,6 +21,105 @@ to your liking. Modifying EpicLoot items works the same as any other(see EXAMPLE
 above).
 ###########################################################################
 
+
+##############################PRICES FORMUALE##############################
+Shields
+Non-Serpent Shields
+n^2.1 n={50:100} 		- where n is the tier adjusted to a fit a min/max
+						of 50 for tier 1 (e.g. Wood), to 100 for BlackMetal
+Serpent Shield (rationale: mid tier with high risk potential)
+n^2.1 n={75} -  where n is adjusted to be between Iron and Silver
+
+
+Chest & Leggings
+n^2 n={40:100} 			- where n is the tier adjusted to a fit a min/max
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Helmets
+n^1.9 n={50:100} 		- where n is the tier adjusted to a fit a min/max
+						of 40 for tier 0 (e.g. Rags), to 100 for BlackMetal
+
+Capes
+n^1.8 n={50,60,90,100} 	- where n is the tier adjusted to a fit a min/max
+						of 50 for tier 1 (e.g. Deer Hide), to 100 for Wolf Cape
+						(rationale: Wolf cape has frost resistance by default
+					 	and is the second most durable cape in the game, if
+					 	enchanted to legedary with maximal affixes it would be
+					 	exceptionally good)
+
+Weapons
+n^2.2 n={70:100} 		- where n is the tier adjusted to a fit a min/max
```

Comment:
Given the serious difference in power between tiers, I feel like this price range is far too tight and too high.

Reply:
Okay, widening the gaps is easy enough, as for the price range I feel like its fine as is though, can discuss with the others?